src/jdk.compiler/share/classes/com/sun/tools/javac/util/Iterators.java

```diff
@@ -43,59 +44,45 @@ public static <I, O> Iterator<O> createCompoundIterator(Iterable<I> inputs, Func
         return new CompoundIterator<>(inputs, converter);
     }
 
-    private static class CompoundIterator<I, O> implements Iterator<O> {
+    private static final class CompoundIterator<I, O> implements Iterator<O> {
```

Comment:
Nit: why final? Seems quite unnecessary for a private class?

Reply:
Reverted in 6cda4b3.